### PR TITLE
Create ui playmusic screen

### PR DESCRIPTION
**Pull request by Hiepsisoma**
https://edu-redmine.framgia.vn/issues/6571
52974592_794369827590709_9186560370376966144_n

### CODE REVIEW

**Comment by Hiepsisoma** [comment]
ready

File: app/src/main/res/layout/activity_play_music.xml [hunk 1]
```diff
@@ -6,4 +6,187 @@
     android:layout_height="match_parent"
     tools:context=".ui.playmusic.PlayMusicActivity">
 
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/tool_bar"
+        android:layout_width="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml]: view con trong constrain ko có match parent

File: app/src/main/res/layout/activity_play_music.xml [hunk 2]
```diff
@@ -6,4 +6,187 @@
...
+        android:layout_height="?attr/actionBarSize"
+        app:layout_collapseMode="pin"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
+        app:theme="@style/ToolbarColoredBackArrow"
+        tools:ignore="MissingConstraints" />
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml:20]: remove bl

File: app/src/main/res/layout/activity_play_music.xml [hunk 3]
```diff
@@ -6,4 +6,187 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/text_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_10"
+        android:layout_marginLeft="@dimen/dp_10"
+        android:text="00:00"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml]: hardcode

File: app/src/main/res/layout/activity_play_music.xml [hunk 4]
```diff
@@ -6,4 +6,187 @@
...
+        android:layout_marginEnd="@dimen/dp_8"
+        android:layout_marginRight="@dimen/dp_8"
+        android:layout_marginBottom="@dimen/dp_18"
+        android:background="@color/trans"
+        android:src="@drawable/ic_repeat"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/image_button_next" />
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml]: remove bl

**Comment by Hiepsisoma** [comment]
ready

**Pull request merged**